### PR TITLE
Added tooltips to various places

### LINKED ISSUE #1569: Tooltip label additions/corrections
**Stardew valley sidebar button.**
Currently: Loadout Stardew Valley
Change to: Stardew Valley

**Add panel button**
Currently: _No tooltip_
Change to: Add a panel for split screen viewing

**Add tab button**
Currently: _No tooltip_
Change to: Add tab

**Settings**
Currently: _No tooltip_
Change to: Settings

**Help button**
Currently: _No tooltip_
Change to: Help and feedback

**Add button (In Library item rows)**
Currently: _No tooltip_
Change to: Add mod to your mod list

**Preview changes**
Currently: _No tooltip_
Change to: View the changes that will be applied to your game folder

**When hovering over individual file changes in the preview view i.e. modified, added, removed**
Currently: _No tooltips_
Added files: File will be added
Modified: File will be modified
Removed: File will be removed

**Apply**
Currently: _No tooltip_
Change to: Apply your changes to the game folder

### PR DESCRIPTION
**Pull request by Al12rs**
- closes #1569

### CODE REVIEW

File: src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonView.axaml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     </Design.DataContext>
     <Button Classes="SpineDownloadButton Idle" x:Name="ParentButton">
         <ToolTip.Tip>
-            <TextBlock x:Name="ToolTipTextBlock"/>
+            <TextBlock Classes="ToolTipText" x:Name="ToolTipTextBlock"/>
```
Comment (erri120) [inline src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonView.axaml:18]: Why is this class needed? If every `TextBlock` inside a `ToolTip` should have this class, then the class shouldn't exist in the first place, and all the styles of the class should be made the default.

File: src/NexusMods.App.UI/Controls/Trees/Files/IFileTreeNodeViewModel.cs [hunk 2]
```diff
@@ -54,17 +54,36 @@ string ToFormattedFileCount()
...
+            {
+                FileChangeType.Added => Language.IFileTreeNodeViewModel_FormattedChangeStateToolTip_Added,
+                FileChangeType.Modified => IsFile
+                    ? Language.IFileTreeNodeViewModel_FormattedChangeStateToolTip_ModifiedFile
+                    : Language.IFileTreeNodeViewModel_FormattedChangeStateToolTip_ModifiedFolder,
+                FileChangeType.Removed => Language.IFileTreeNodeViewModel_FormattedChangeStateToolTip_Removed,
+                FileChangeType.None => string.Empty,
+            };
+        }
     }
```
Comment (erri120) [inline src/NexusMods.App.UI/Controls/Trees/Files/IFileTreeNodeViewModel.cs:87]: Please make two methods for this. Huge `get` blocks shouldn't exist.

File: src/NexusMods.App.UI/Controls/Trees/FileTreeView.axaml [hunk 3]
```diff
@@ -37,6 +37,13 @@
                 <DataTemplate x:Key="FileNameColumnTemplate" DataType="{x:Type files:IFileTreeNodeViewModel}">
                     <files:FileTreeNodeView DataContext="{CompiledBinding}" />
                 </DataTemplate>
+                <DataTemplate x:Key="FileStateColumnTemplate" DataType="{x:Type files:IFileTreeNodeViewModel}">
+                    <TextBlock Text="{CompiledBinding FormattedChangeState}" >
+                        <ToolTip.Tip>
+                            <TextBlock Text="{CompiledBinding FormattedChangeStateToolTip}" />
```
Comment (erri120) [inline src/NexusMods.App.UI/Controls/Trees/FileTreeView.axaml:43]: Missing `ToolTipText` class.

**Review by erri120 — CHANGES_REQUESTED** [review]

File: src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonView.axaml (hunk 1 above)
Comment (erri120) [reply src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonView.axaml:18]: Tooltips should never contain anything other than text and maybe an icon. Putting a button in a tooltip is pure evil.

File: src/NexusMods.App.UI/Controls/Trees/Files/IFileTreeNodeViewModel.cs (hunk 2 above)
Comment (Al12rs) [reply src/NexusMods.App.UI/Controls/Trees/Files/IFileTreeNodeViewModel.cs]: I still need the properties to bind to from XAML, but refactored away the get contents into methods

File: src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonView.axaml (hunk 1 above)
Comment (erri120) [reply src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonView.axaml:18]: For `ToolTip` specifically, just do `ToolTip TextBlock`.

Comment (Al12rs) [reply src/NexusMods.App.UI/Controls/Spine/Buttons/Download/SpineDownloadButtonView.axaml:18]: In MO2 we have tables in there, just saying

**Review by erri120 — APPROVED** [review]

**Pull request merged**